src/Core/src/Primitives/MathF.cs

```diff
@@ -0,0 +1,35 @@
+#if NETSTANDARD2_0
+namespace System;
+
+internal static class MathF
```

Comment:
I think you should also be able to just put the type in `src/Core/src/Core.cpsroj` and the other projects can use it via `InternalsVisibleTo`? I don't think you should need the file in here twice. Thanks!